dyninstAPI_RT/src/RTwinnt.c

```diff
@@ -361,8 +364,8 @@ static struct trap_mapping_header *getStaticTrapMap(unsigned long addr, unsigned
                  + curSecn.SizeOfRawData 
                  - 16);
    if (0 != strncmp("DYNINST_REWRITE", str, 15)) {
-      fprintf(stderr, "ERROR IN RTLIB: getStaticTrapMap found bad string [%s] at %lx %s[%d]\n", 
-              str, str, __FILE__,__LINE__);
```

Comment:
This should be [%s] at %p (%s[%d]) to capture the string as a string and as a pointer for debugging.